### PR TITLE
Optimizations

### PR DESCRIPTION
**Pull request by kckennylau**

### CODE REVIEW

**Comment by Mego** [comment]
Make the change on line 731 like I mentioned in the comment, and this should be good to merge, assuming the CI build succeeds.

**Comment by kckennylau** [comment]
I've changed `from copy import copy as _copy` to `from copy import deepcopy as _copy`.

File: seriously/SeriouslyCommands.py [hunk 1]
```diff
@@ -728,7 +728,7 @@ def dupe_each_n_fn(srs):
     tmp = []
     while srs.stack:
         b = srs.pop()
-        tmp = [b]*a + tmp
+        tmp = [copy.deepcopy(b) for _ in range(a)] + tmp
```
Comment (Mego) [inline seriously/SeriouslyCommands.py]: This wasn't necessary - `copy` in the global scope is a wrapper around `copy.copy` (well, now, `copy.deepcopy`) that properly handles generators.

**Comment by kckennylau** [comment]
I just noticed that you have an `as_list` function that you wrote yourself. Should I replace all occurrences of `list` with `as_list`?

**Comment by kckennylau** [comment]
Actually, I shouldn't. `as_list` wraps non-iterables.

**Comment by Mego** [comment]
No, `as_list` deeply converts iterables to lists. It's very expensive, and only really needed when working with generators that can yield other generators.

**Pull request merged**